test-dts/component.test-d.ts

```diff
@@ -190,7 +191,7 @@ describe('object props', () => {
     expectType<Number>(rawBindings.setupA)
     expectType<Ref<Number>>(rawBindings.setupB)
     expectType<Ref<Number>>(rawBindings.setupC.a)
-    expectType<Number>(rawBindings.setupA)
```

Comment:
was this one removed by accident?

Reply:
No, it was added by accident 😅  it's a dupe of line `191`